### DISCUSSION TITLE
Is there any api configuration that changes the matching file suffix?

### DISCUSSION THREAD

**Discussion by zhaohuanyuu** [mallowigi/a-file-icon-idea#336] [Ideas]
**problem**
gatsby-**.js|ts supported esm module now, The file suffix of .mjs/ts cannot match the corresponding icon

**request**
Is there any api configuration that changes the matching file suffix? to set regexp to match the icon

![image](https://user-images.githubusercontent.com/25791574/207766107-8d319f57-30de-4c09-a18c-f3a3f4b233e6.png)

**Comment by mallowigi** [comment]
Dont use discussions for issues!
